src/data/faq.json

```diff
@@ -1333,6 +1333,35 @@
+                            "<b>Health departments are used to capacity:</b><br/>Currently, COVID-19 infection numbers are at very high levels and a high number of positive test results are being submitted through the Corona-Warn-App. Therefore, please understand that the health department cannot serve all contacts. Let the health department handle its important tasks, so contact your health department only if you absolutely need assistance, for example, because your employer or educational institution requires a certificate.",
+                            "<b>Recommendations for action:</b>",
+                            "<ul><li>Get tested. If you receive a warning of increased risk (\"red tile\"), you are entitled to a free test. This does not necessarily have to be a PCR test.</li><li>A minimum of 7 days, preferably 10 days, is recommended for the period of self-quarantine from the date of the last increased risk encounter. Refer to the red tile for the date of the last increased risk encounter.</li><li>If possible, refrain from visits and activities where you will be with other people. Have groceries delivered or ask someone to do the shopping; avoid public transportation and refrain from air and train travel.</li><li>In particular, avoid contact with vulnerable persons and groups, such as the elderly and those with pre-existing conditions.</li><li>Also minimize contacts in your own household.</li><li>If your employer gives you the option, work from home.</li><li>Watch for symptoms of illness for at least 14 days (from the date of the increased risk encounter) and contact your primary care physician's office if you feel ill or experience symptoms.</li><li>Follow the rules of hygiene.</li></ul>",
+                            "For more helpful information on home quarantine, click here: <a href='https://www.rki.de/covid-19-quarantaene' target='_blank'>Home quarantine (ordered by the health department): flyer for contacts. (German only)</a><br/>More information on quarantine measures can be found here: <a href='#quarantine_measures'>https://www.coronawarn.app/en/faq/#quarantine_measures</a><br/>Find testing options <a href='#where_can_i_get_tested'>Where can I get tested?</a><br/>More information on the testing process: <a href='#red_card_how_to_test'>Red Card? How to get the Corona Test</a>"

```

Comment:
Instead of using a literal URL in:
`More information on quarantine measures can be found here: <a href='#quarantine_measures'>https://www.coronawarn.app/en/faq/#quarantine_measures</a>`

suggest
`Read the FAQ article <a href='#quarantine_measures'>Quarantine measures</a> for more information.`

Reply:
@MikeMcC399 OK! Thanks.